src/containers/ApiInit.tsx

```diff
@@ -0,0 +1,31 @@
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
This file will handle listening for events emitted by the server. Just thinking some ore on it, to avoid this file from getting too large, should move the handler function to a saga so this component will just listen for the event and fire and essentially convert it to a redux action call, which a saga will handle.